server/src/main/models/index.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs: any        = require('fs');
+var path: any      = require('path');
+var Sequelize: any = require('sequelize');
```

Comment:
I'm not sure what the common conventions for this are, but doesn't it make more sense to name the imported module "sequelize" (lowercase S) and the variable `var sequelize = ...` something that stands out better to avoid confusion, for example "dbConnection" or something like that.